packages/expo-media-library/ios/MediaLibraryUtilities.swift

```diff
@@ -218,7 +218,7 @@ func exportCollection(_ collection: PHAssetCollection?, folderName: String? = ni
     "assetCount": assetCountOfCollection(collection),
     "startTime": exportDate(collection.startDate),
     "endTime": exportDate(collection.endDate),
-    "approximateLocation": exportLocation(location: collection.approximateLocation),
+    "approximateLocation": exportLocation(location: collection.approximateLocation) ?? NSNull(),
```

Comment:
```suggestion
    "approximateLocation": exportLocation(location: collection.approximateLocation) ?? nil,
```
Any reason we wouldn't use `nil` here?

Reply:
I think we should, the code would be a lot more readable